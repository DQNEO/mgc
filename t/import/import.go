package main

import (
	"github.com/DQNEO/minigo/t/import/mylib"
	"fmt"
)

func f1() {
	mylib.MyFunc()
	fmt.Printf("%d\n", mylib.MyNumber)
	fmt.Printf("%s\n", mylib.MyString)
}

func main() {
	f1()
}
