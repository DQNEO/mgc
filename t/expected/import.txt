Hello MyFunc
1234
MyString
